stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/filepreview/MediaAttachmentPreviewHandler.kt

```diff
@@ -42,10 +40,30 @@ public class MediaAttachmentPreviewHandler(private val context: Context) : Attac
         )
     }
 
-    private companion object {
-        private const val MIME_TYPE_VIDEO = "video"
-        private const val MIME_TYPE_AUDIO = "audio"
-        private const val MIME_SUBTYPE_MP4 = "mp4"
-        private const val MIME_SUBTYPE_QUICKTIME = "quicktime"
-    }
+    private fun buildSubTypeList() = listOf(
```

Comment:
This is a fallback option and as such is not guaranteed to cover all formats in all variations as the list of such would be prohibitively long. 

We might partially mitigate this if we better structure mime types inside `ModelType` as previously mentioned in our discussions.